spec/controllers/distributions_controller_spec.rb

```diff
@@ -18,6 +18,7 @@
             organization_name: organization.id,
             distribution: {
               partner_id: partner.id,
+              issued_at: Date.yesterday,
```

Comment:
Why do we need these specifications when you added it to the factory already?

Or should `yesterday` be the default?

Reply:
I added values to the factories so creating these records will always have an `issued_at` field, but here these are parameters for a request. If I omit this parameter we are trying to create a distribution with no `issued_at` field which is now invalid after these changes.

When you go to create a new distribution (although same for purchase/donation), a default value for `issued_at` is displayed. I could add a default value for `issued_at` in the controller, but then we are in a similar place as before (someone emptying the field, clicking save, and unexpected behavior occurring).

If someone decides to empty the issue date field, I think it would be better to respond with a validation error and make them put in a date. But that's just an assumption on my part based on the issue description, so I can change if you think coercing missing issue dates into a default would be better.

The date `yesterday` is arbitrary here, it could be any valid date/time.